src/cook-web/src/components/billing/BillingPlanComparisonTable.module.scss

```
@@ -0,0 +1,277 @@
+  padding: 2px 8px;
+  border-radius: 999px;
+  background: linear-gradient(135deg, #ef4444 0%, #dc2626 100%);
+  color: $color-on-accent;
+  white-space: nowrap;
+  @include font-badge;
+}
+
+.columnBadgeIcon {
+  width: 12px;
+  height: 12px;
+  fill: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)